iotedgedev/utility.py

```diff
@@ -40,6 +40,13 @@ def call_proc(self, params, shell=False, cwd=None):
         except Exception as e:
             self.output.error("Error while executing command: {0}. {1}".format(' '.join(params), str(e)))
 
+    def check_dependency(self, params, description, shell=False):
+        try:
+            self.exe_proc(params, shell=shell)
+        except:
+            self.output.error("The {0} is required by the Azure IoT Edge Dev Tool. For installation instructions, see the README at https://aka.ms/iotedgedev.".format(description))
```

Comment:
I like that this is done in a check_dependency utility now, and thanks for updating the instructions to go with it! 

One piece of feedback I got on the original message here is that the message should be clear that {tool} is only required if the user is trying to build {lang/platform} modules. 

Maybe consider dropping "The " from the front, so the format string is:
`"{0} is required by the Azure IoT Edge Dev Tool. For installation instructions, see the README at https://aka.ms/iotedgedev."`

And then the description passed in can be of the form
`"To build python modules, the Cookiecutter tool "`
or
`"To build .NET modules, the .NET Core SDK "`

etc.

Reply:
Thanks for the information! I have adopted your awesome proposal.